emails/templates/emails/disabled_mask_for_spam.html

```diff
@@ -0,0 +1,32 @@
+{% comment %}
+  Note that Django only loads strings from some Fluent files.
+  See privaterelay/ftl_bundles.py.
+{% endcomment %}
+{% load ftl %}
+{% load email_extras %}
+{% withftl bundle='privaterelay.ftl_bundles.main' language=language %}
+
+{% include "emails/direct_email_header.html" %}
+
+    <table role="presentation" border="0" cellpadding="0" cellspacing="10px" style="padding: 60px 30px 120px 30px;" align="center">
+        <tr>
+            <td style="max-width:850px; padding-top: 0px; padding-bottom: 0px; text-align: center;">
+                {% with mask|striptags|urlencode as mask_url %}

```

Comment:
The honest answer is copy-pasta from [this line](https://github.com/mozilla/fx-private-relay/blob/2e576168efeb946737fd0da0ee7c07ca3742588d/emails/templates/emails/wrapped_email.html#L134) of `wrapped_email.html` The answer for that one is probably to prevent XSS if someone were to put XSS into the localpart of a domain address? Highly unlikely, but better safe than sorry?